force-app/main/default/lwc/dataTableWrapper/__tests__/dataTableWrapper.test.js

```diff
@@ -16,7 +16,7 @@ describe("c/dataTableWrapper", () => {
         component = null;
     });
 
-    it("FIXME should get code coverage for handleRowChange", () => {
+    it("should get code coverage for handleRowChange", () => {
```

Comment:
@a-contreras 
Can you change this to be more descriptive of what is expected from this test? Something like `"should dispatch draftvaluechange on row change"`

Reply:
Sure!